dom/events/delegate/derive-enter-leave.js

```diff
@@ -0,0 +1,48 @@
+'use strict';
+
+var domEvents = require("../events");
+
+// Some mouse/pointer events do not bubble so we derive these events from other
+// bubbling events so they work with delegated listeners
+
+var eventMap = {
+    mouseenter: "mouseover",
+    mouseleave: "mouseout",
+    pointerenter: "pointerover",
+    pointerleave: "pointerout"
+  },
+  _addDelegateListener = domEvents.addDelegateListener,
+  _removeDelegateListener = domEvents.removeDelegateListener;
+
+
+domEvents.addDelegateListener = function(eventType, selector, handler) {
+  if (eventMap[eventType] !== undefined) {
+    var origHandler = handler;
+
+    this._derivedHandlers = this._derivedHandlers || {};
```

Comment:
Should I keep the reference to the modified handler elsewhere? Use can-util/dom/data insetad?